Packs/PingIdentity/Integrations/PingOne/README.md

```diff
@@ -0,0 +1,412 @@
+Integrates with the PingOne Management API to unlock, create, delete and update users.  
+This integration was integrated and tested with version xx of PingOne
+
+## Configure PingOne on Cortex XSOAR
+
```

Comment:
Please generate a new Readme file once you edit the yml file (add the missing commands configuration).

Reply:
Readme now included.